### PR TITLE
Added description BraintreeInput

### PR DESCRIPTION
**Pull request by sedonik**
### Description (*)

Braintree payment method input has multiple fields like: device_data, is_active_payment_token_enabler, payment_method_nonce which don't have the description.

### Issue

1. magento/graphql-ce#876: [Braintree] Add description to BraintreeInput

### CODE REVIEW

**Review by lenaorobei — CHANGES_REQUESTED** [review]
The description should provide clear understanding of the field purpose.

File: app/code/Magento/BraintreeGraphQl/etc/schema.graphqls [hunk 1]
```diff
@@ -11,9 +11,9 @@ input PaymentMethodInput {
 }
 
 input BraintreeInput {
-    payment_method_nonce: String!
-    is_active_payment_token_enabler: Boolean!
-    device_data: String
+    payment_method_nonce: String! @doc(description:"Required input for Braintrees payments")
```
Comment (YevSent) [inline app/code/Magento/BraintreeGraphQl/etc/schema.graphqls]: Unfortunately, the added description doesn't explain the field purpose.
The `payment_method_nonce` is a one-time payment token generated by Braintree payment gateway based on card details and this token is required to make sale transaction.

File: app/code/Magento/BraintreeGraphQl/etc/schema.graphqls [hunk 2]
```diff
@@ -11,9 +11,9 @@ input PaymentMethodInput {
...
+    is_active_payment_token_enabler: Boolean! @doc(description:"Required input for Braintrees payments")
```
Comment (YevSent) [inline app/code/Magento/BraintreeGraphQl/etc/schema.graphqls]: This field is required only if Vault is enabled for Braintree payment integration. The `is_active_payment_token_enabler` shows if an entered by a customer credit/debit card should be tokenized for later usage.

File: app/code/Magento/BraintreeGraphQl/etc/schema.graphqls [hunk 3]
```diff
@@ -11,9 +11,9 @@ input PaymentMethodInput {
...
+    device_data: String @doc(description:"Additional information for Braintrees payments")
```
Comment (YevSent) [inline app/code/Magento/BraintreeGraphQl/etc/schema.graphqls]: The `device_data` will be presented only in a case if Kount (advanced fraud protection) is enabled for Braintree payment integration. The device data contains a fingerprint provided by Braintree JS SDK and should be sent with sale transaction details to the Braintree payment gateway.

**Review by YevSent — CHANGES_REQUESTED** [review]

**Comment by sedonik** [comment]
@lenaorobei, @joni-jones 
Descriptions are updated. 
Could you check?

File: app/code/Magento/BraintreeGraphQl/etc/schema.graphqls [hunk 4]
```diff
@@ -11,9 +11,9 @@ input PaymentMethodInput {
 }
 
 input BraintreeInput {
-    payment_method_nonce: String!
-    is_active_payment_token_enabler: Boolean!
-    device_data: String
+    payment_method_nonce: String! @doc(description:"The one-time payment token generated by Braintree payment gateway based on card details. Required field to make sale transaction.")
+    is_active_payment_token_enabler: Boolean! @doc(description:"This field states whether an entered by a customer credit/debit card should be tokenized for later usage. Required only if Vault is enabled for Braintree payment integration.")
```
Comment (lenaorobei) [inline app/code/Magento/BraintreeGraphQl/etc/schema.graphqls]:
```suggestion
    is_active_payment_token_enabler: Boolean! @doc(description:"States whether an entered by a customer credit/debit card should be tokenized for later usage. Required only if Vault is enabled for Braintree payment integration.")
```

File: app/code/Magento/BraintreeGraphQl/etc/schema.graphqls [hunk 5]
```diff
@@ -11,9 +11,9 @@ input PaymentMethodInput {
...
+    device_data: String @doc(description:"This field contains a fingerprint provided by Braintree JS SDK and should be sent with sale transaction details to the Braintree payment gateway. The device_data will be presented only in a case if Kount (advanced fraud protection) is enabled for Braintree payment integration.")
```
Comment (lenaorobei) [inline app/code/Magento/BraintreeGraphQl/etc/schema.graphqls]:
```suggestion
    device_data: String @doc(description:"Contains a fingerprint provided by Braintree JS SDK and should be sent with sale transaction details to the Braintree payment gateway. Should be specified only in a case if Kount (advanced fraud protection) is enabled for Braintree payment integration.")
```

**Review by lenaorobei — CHANGES_REQUESTED** [review]
Please see my suggestions.

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5859](https://jira.corp.magento.com/browse/ENGCOM-5859) has been created to process this Pull Request
:eight_spoked_asterisk: @lenaorobei, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Pull request merged**